presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -269,6 +275,10 @@ void PrestoServer::run() {
       std::make_unique<SystemPrestoToVeloxConnector>("system"));
   registerPrestoToVeloxConnector(
       std::make_unique<SystemPrestoToVeloxConnector>("$system@system"));
+#ifdef PRESTO_ENABLE_TPCDS_CONNECTOR
```

Comment:
The TPC-H connector option for Velox is present in the Velox CMakeLists.txt.
`VELOX_ENABLE_TPCH_CONNECTOR`
Since it is ON by default in Velox, it works for Prestissimo. We must fix this.
Having these options reduces build time and avoids dependencies in other cases. We can get a light worker and these options don't hurt.
The ifdef can go to the TPC-DS implementation and keep the usage here clean. See how Parquet and storage adapters are handled.